### PR TITLE
docs: updates readme

### PR DESCRIPTION
**Pull request by dzucconi**
Re: https://github.com/artsy/palette/pull/1234

### CODE REVIEW

**Comment by dzucconi** [comment]
Hm, trying to re-trigger Circle. All my auth got revoked yesterday.

File: README.md [hunk 1]
```diff
@@ -5,10 +5,13 @@ Artsy's Design System
 ## Meta
 
 - Storybook: [https://palette-storybook.artsy.net](https://palette-storybook.artsy.net)
-- Docs: [https://palette.artsy.net](https://palette.artsy.net)
-- Docs Github Repo: [https://github.com/artsy/palette-docs](https://github.com/artsy/palette-docs)
+- <del>Docs: [https://palette.artsy.net](https://palette.artsy.net)</del> (Deprecated: Documentation will be moved to Storybook to have a single source of truth.)
```
Comment (damassi) [inline README.md:8]: 👍

File: packages/palette/src/svgs/_icons.story.tsx [hunk 2]
```diff
@@ -21,29 +23,49 @@ export default {
...
-    </GridColumns>
+    <>
+      <Message variant="warning">
+        Please see{" "}
+        <a
+          href="https://github.com/artsy/icons"
+          target="_blank"
+          rel="noreferrer"
+        >
+          @artsy/icons
```
Comment (damassi) [inline packages/palette/src/svgs/_icons.story.tsx:34]: 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette-charts@26.1.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette-charts%4026.1.0), [`@artsy/palette@27.1.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4027.1.0) :rocket:

**Pull request merged**